packages/client/components/SpotlightModal.tsx

```diff
@@ -183,8 +186,28 @@ const SpotlightModal = (props: Props) => {
   const {viewer} = data
   const {meeting} = viewer
   const phaseRef = useRef(null)
-  if (!meeting) return null
-  const {spotlightReflection} = meeting
+  const spotlightReflectionIds = useRef<null | string[]>(null)
+  const spotlightGroup = meeting?.spotlightGroup
+  const firstReflectionId = spotlightGroup?.reflections[0]?.id
+  // const secondReflectionId = spotlightGroup?.reflections[1]?.id
+  const showSpotlight = meeting && spotlightGroup
+
+  useEffect(() => {
+    if (!spotlightGroup) return
+    const {current: ids} = spotlightReflectionIds
+    if (!ids && firstReflectionId) {
+      spotlightReflectionIds.current = [firstReflectionId]
+    }
+    // TODO: uncomment for groups -> source issue
+    // else if (firstReflectionId && secondReflectionId && ids?.includes(secondReflectionId)) {
+    //   spotlightReflectionIds.current = [...ids, firstReflectionId]
+    // }
+    else if (!firstReflectionId || !ids?.includes(firstReflectionId)) {
+      setTimeout(() => {
```

Comment:
It we end up using `setTimeout` it'd be nice to call `clearTimeout` in `useEffect` cleanup